### PR TITLE
Added table of contents on index page

### PR DESCRIPTION
**Pull request by eugeneyak**

### CODE REVIEW

**Comment by jarosluv** [comment]
Thanks for PR. But before we can merge it, we need to think about two points:
1. Can we auto-generate table of contents in Jekyll theme?
2. How we can refine table of contents layout? May be add it in modal window and open it with toggle button in bottom navigation panel?

File: index.md [hunk 1]
```diff
@@ -3,6 +3,42 @@ title: Документация Elixir
...
+* [Протоколы](/docs/protocols.html)
+* [Списковые выражения](/docs/comprehensions.html)
+* [Сигилы](/docs/sigils.html)
+* [try, catch и rescue](/docs/try-catch-and-rescue.html)
+* [Спецификации типов и поведения](/docs/typespecs-and-behaviours.html)
+* [Библиотеки Эрланга](/docs/erlang-libraries.html)
+* [Куда двигаться дальше](/docs/where-to-go-next.html)
+* [Введение в Mix](/docs/introduction-to-mix.html)
+* [Агент](/docs/agent.html)
+* [GenServer](/docs//genserver.html)
```
Comment (rtrv) [inline index.md:30]: Double slash in `/docs//genserver.html`

**Review by rtrv — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (jarosluv) [reply index.md:30]: This is completely wrong url to be exact.

**Comment by eugeneyak** [comment]
The table of contents can be done automatically by Jekyll, but the question is the correctness of the links and the order of the chapters:

```
{% for p in site.pages %}
* [{{ p.title }}](/docs{{ p.url }})
{% endfor %}
```

**Pull request merged**